ApollosProject/apollos-apps
# feat(follow): update follow language

## ✏️ Description
- Updates language to follow a user to better reflect flow


## 🔬 To Test
1. Go to individual prayer
2. Tap three vertical dot button
3. Should see `Request to follow [User]` in the menu

## 📸 Screenshots
![image](https://user-images.githubusercontent.com/7557597/210882559-668b46bc-a216-4687-8bd7-20fecc2db3c1.png)



## ⚠️ To-do Before Merge



## 📝 Checklist
